The tokenizer process does not correctly handle paths with one or more spaces

## Description

The tokenizer process does not correctly handle paths with one or more spaces.

The output of the process is:
```
INFO: Sensor TokenizerSensor [ps]
##[error]D:\4\_work\3\s\.scannerwork\.sonartmp\ps4136015749188592249parser.ps1 : Cannot process argument transformation on 
parameter 'depth'. Cannot convert value "part-of-directory-after-space\file.ps1" to type "System.Int32". Error: "Input
D:\4\_work\3\s\.scannerwork\.sonartmp\ps4136015749188592249parser.ps1 : Cannot process argument transformation on 
parameter 'depth'. Cannot convert value "part-of-directory-after-space\file.ps1" to type "System.Int32". Error: "Input 
##[error]string was not in a correct format."
```

## Environment
OS: Windows 2016 v1607
Packages and versions:
PowerShell	5.1.14393.2636
java	C:\Program Files\Java\jre1.8.0_161
java_8_x64	C:\Program Files\Java\jre1.8.0_161
PSScriptAnalyzer	1.17.1
SonarQube 6.7.6
sonar-ps-plugin 0.3.2

--- a/sonar-ps-plugin/src/main/java/org/sonar/plugins/powershell/sensors/TokenizerSensor.java
+++ b/sonar-ps-plugin/src/main/java/org/sonar/plugins/powershell/sensors/TokenizerSensor.java
@@ -86,7 +86,7 @@ public class TokenizerSensor extends BaseSensor implements org.sonar.api.batch.s
 		for (final InputFile inputFile : inputFiles) {
 			try {
 
-				final String analysisFile = inputFile.file().getAbsolutePath();
+				final String analysisFile = String.format("'%s'", inputFile.file().getAbsolutePath());
 
 				// skip reporting temp files
 				if (analysisFile.contains(".scannerwork")) {

